docs/user/resources/02-logpipeline.md

```diff
@@ -164,9 +164,12 @@ For details, see the [LogPipeline specification file](https://github.com/kyma-pr
 | Parameter | Type | Description |
 | ---- | ----------- | ---- |
 | **conditions**  | \[\]object | An array of conditions describing the status of the pipeline. |
-| **conditions.&#x200b;lastTransitionTime**  | string | An array of conditions describing the status of the pipeline. |
-| **conditions.&#x200b;reason**  | string | Reason of last transition. |
-| **conditions.&#x200b;type**  | string | The possible transition types are:<br>- `Running`: The instance is ready and usable.<br>- `Pending`: The pipeline is being activated. |
+| **conditions.&#x200b;lastTransitionTime** (required) | string | lastTransitionTime is the last time the condition transitioned from one status to another. This should be when the underlying condition changed.  If that is not known, then using the time when the API field changed is acceptable. |
+| **conditions.&#x200b;message** (required) | string | message is a human readable message indicating details about the transition. This may be an empty string. |
```

Comment:
In which cases is it okay that the the human-readable message is empty?

Reply:
It is not a good practise for sure to have an empty string as the message, but the [`metav1.Condition`](https://github.com/kubernetes/apimachinery/blob/v0.29.1/pkg/apis/meta/v1/types.go#L1497) allows it. This documentation is auto-generated from the upstream package https://github.com/kubernetes/apimachinery/blob/v0.29.1/pkg/apis/meta/v1/types.go#L1497 and we should not manually change it here.

Reply:
oha, alright